### PR TITLE
Fix getBounds to return Bounds (not accidentally treat right edge as width); fix container to optionally take size params

### PR DESCRIPTION
**Pull request by lackhand**

### CODE REVIEW

File: src/gameobjects/container/Container.ts [hunk 1]
```diff
@@ -6,9 +6,13 @@ export class Container extends GameObject implements IContainer
 {
     protected _alpha: number = 1;
 
-    constructor (x: number = 0, y: number = 0)
+    constructor (x: number = 0, y: number = 0, width: number = undefined, height: number = undefined)
```
Comment (photonstorm) [inline src/gameobjects/container/Container.ts:9]: Should be `width?: number, height?: number`

Comment (lackhand) [reply src/gameobjects/container/Container.ts]: Done, thanks!

**Comment by lackhand** [comment]
please take a look, thanks!

**Comment by lackhand** [comment]
Bounds is going away!

**Pull request closed without merging**